components/ConfirmationModal.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledButton from './StyledButton';
+import Container from './Container';
+import { P } from './Text';
+import Modal, { ModalBody, ModalHeader, ModalFooter } from './StyledModal';
+
+const ConfirmationModal = ({ show, header, body, onClose, actions }) => (
+  <Modal width="570px" show={show} onClose={() => onClose()}>
```

Comment:
Can directly pass the function

```suggestion
  <Modal width="570px" show={show} onClose={onClose}>
```